debezium-storage/debezium-storage-jdbc/src/main/java/io/debezium/storage/jdbc/JdbcCommonConfig.java

```diff
@@ -37,7 +37,7 @@ public class JdbcCommonConfig {
 
     public static final Field PROP_PASSWORD = Field.create(CONFIGURATION_FIELD_PREFIX_STRING + "password")
             .withDescription("Password of the database which will be used to access the database storage")
-            .withValidation(Field::isRequired);
+            .withValidation(Field::isOptional);
```

Comment:
Why this change? It seems unrelated.

Reply:
Hi @jpechane , sorry its not related, its just that there might be cases where you might not have to provide a password, especially connecting to local , dev databases. 
Should I remove it?